.travis.yml

```diff
@@ -17,6 +17,9 @@ addons:
       - postgresql-client-10
       - libgconf-2-4  # for Cypress
 
+before_install:
+  - sudo apt-get install apache2-dev
```

Comment:
What is this for?

Reply:
mod_wsgi: https://modwsgi.readthedocs.io/en/develop/user-guides/quick-installation-guide.html

Reply:
(since we install it via pip now and thus it gets installed on aws)